### DISCUSSION TITLE
Muted mod makes the game become laggy after 2-3mins of playing

### DISCUSSION THREAD

**Discussion by anispwyn** [ppy/osu#17810] [Q&A]
### Type

Performance

### Bug description

Game become laggy when i tried to play with relax, hidden & muted mod

**u can ignore this boring story**
So i was playing normally with relax & hidden and its run fine
And then i want to try some fun mods and i found Muted
After i try to play it with relax, hidden & muted i detect some HUGE performance issue

I dont even know what i do to cause my game to be this laggy (singlethread)
So i try to run multithread and the results still the same

(the video is 17 min long and u can see sometimes my cursor just stop moving or laggy cursor) 

Device info

![IMG_20220414_022958](https://user-images.githubusercontent.com/36318840/163255901-819c677f-eadc-4e97-93f6-b9f8c2f7a1b5.jpg)
![IMG_20220414_022944](https://user-images.githubusercontent.com/36318840/163255904-612ca500-20ed-4454-9ed0-60d0fb296257.jpg)
![IMG_20220414_022818](https://user-images.githubusercontent.com/36318840/163255911-e87f7fdc-442b-4071-97d2-c5963ed4a73d.jpg)

### Screenshots or videos

https://youtu.be/9hDqN8-3Lhw
(at 2:32 april 14 the resolution still 360p just wait for a couple hours for youtube to finish processing the high res) 

### Version

2022.409.0

### Logs

[database.log](https://github.com/ppy/osu/files/8484123/database.log)
[network.log](https://github.com/ppy/osu/files/8484126/network.log)
[performance.log](https://github.com/ppy/osu/files/8484127/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8484128/runtime.log)

**Comment by frenzibyte** [comment]
> I dont even know what i do to cause my game to be this laggy (singlethread)
So i try to run multithread and the results still the same

Running on multi-threaded mode can only ever make it worse for you, as it's not the best option for mobile devices.

In general, mobile performance is not a priority, as desktop comes first and foremost.

**Reply by anispwyn** [reply]
Yeah i know that running multithread on android is a big mistake but
This does happen even i'm on singlethread

**Reply by anispwyn** [reply]
> Please specify a timestamp at which this lag is clear. All I'm seeing is a 17 minutes clip of a regular replay session. Nothing to deduce from it at all.

I'm sorry about the timestamp but it happen everytime (in game not replay) 
So i decided. U know what let me just record my gameplay with & without Muted mods

with mods : https://streamable.com/d4mh9l
without mods : https://streamable.com/bd4xu5 

Logs 
[database.log](https://github.com/ppy/osu/files/8484592/database.log)
[network.log](https://github.com/ppy/osu/files/8484593/network.log)
[performance.log](https://github.com/ppy/osu/files/8484594/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8484595/runtime.log)

**Comment by smoogipoo** [comment]
I can't repro this, nor can I see how it could occur from reading the code. Maybe something Android/Xamarin-specific...?
